bindings/py/setup.py

```diff
@@ -71,6 +71,12 @@ def findRequirements(platform):
 
 
 
+def runTests(args):
+  errno = subprocess.call(["py.test", " ".join(args)])
```

Comment:
It is not really safe to assume that all users running tests are going to have `py.test` on their path. This is sometimes an extra step outside of the normal `pip install pytest` process. Is there some way you can check for that and print a suggestion for devs to install py.test properly?  cc @oxtopus